src/services/role/model.js

```diff
@@ -28,15 +15,15 @@ const roleSchema = new Schema({
 });
 
 roleSchema.methods.getPermissions = function getPermissions() {
-	return roleModel.resolvePermissions([this._id]); // fixme
+	return RoleModel.resolvePermissions([this._id]); // fixme
 };
 
 roleSchema.statics.resolvePermissions = function resolvePermissions(roleIds) {
 	const processedRoleIds = [];
 	const permissions = new Set();
 
 	function resolveSubRoles(roleId) {
-		return roleModel.findById(roleId) // fixme
+		return RoleModel.findById(roleId) // fixme
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['RoleModel' was used before it was defined.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5258109)